packages/map/src/vaadin-map.js

```diff
@@ -4,6 +4,7 @@
  * This program is available under Commercial Vaadin Developer License 4.0, available at https://vaadin.com/license/cvdl-4.0.
  */
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
+import { Attribution, Zoom } from 'ol/control';
```

Comment:
```suggestion
import Attribution from 'ol/control/Attribution';
import Zoom from 'ol/control/Zoom';
```
Maybe we should avoid importing the barrel module. That might lead to importing all the other controls even if they are not used. Kind of depends on how well the bundler (webpack, vite) as well as OL support tree-shaking.